cmd/helm/testdata/output/template-with-invalid-yaml-debug.txt

```diff
@@ -1,13 +1,13 @@
 ---
 # Source: chart-with-template-with-invalid-yaml/templates/alpine-pod.yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: "RELEASE-NAME-my-alpine"
-spec:
-  containers:
-  - name: waiter
-    image: "alpine:3.9"
-    command: ["/bin/sleep","9000"]
-invalid
+ 1 apiVersion: v1
+ 2 kind: Pod
+ 3 metadata:
+ 4   name: "RELEASE-NAME-my-alpine"
+ 5 spec:
+ 6   containers:
+ 7   - name: waiter
+ 8     image: "alpine:3.9"
+ 9     command: ["/bin/sleep","9000"]
+10 invalid
 Error: YAML parse error on chart-with-template-with-invalid-yaml/templates/alpine-pod.yaml: error converting YAML to JSON: yaml: line 11: could not find expected ':'
```

Comment:
The error is on line 10, yet the error message states it's on line 11. Is that to be expected here?

Reply:
Looks like this is related to the yaml package. I assume, when parsing in Unmarshaler, it adds "---" to the start of yaml resource (If I add "---" to the template, it points to the correct line. 

I'll look further into how to fix this.

Reply:
Looks like this is an issue in go-yaml fixed in v3 https://github.com/go-yaml/yaml/issues/542